### DISCUSSION TITLE
No themes and tutor in the git version.

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#1198] [General] [answered]
```
hx --version
helix v0.5.0-171-gc08d2fa
```
I installed the git version. I had to update to 2021 version of rust. I don't find any themes and starting vim tutor opens a blank file named scratch.

**Comment by kajuburfi** [comment, marked as answer]
You have to copy the `runtime` directory (found in the root of `Helix`), to `XDG_CONFIG_HOME` (i.e. `~/.config/helix/`).

Then you will be able to find all themes and the tutor as well.
